sdk/network/Azure.ResourceManager.Network/tests/Tests/UnitTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Text.Json;
+using Azure.ResourceManager.Network.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Network.Tests
+{
+    public class UnitTest
+    {
+        // This is the test for fix of issue: https://github.com/Azure/azure-sdk-for-net/issues/46767
+        [Test]
+        public void DeserializeChangeNumber()
+        {
+            var sr = new StreamReader(Path.Combine("TestData", "ServiceTags.json"));
```

Comment:
```suggestion
            using var sr = new StreamReader(Path.Combine("TestData", "ServiceTags.json"));
```
both of the types here are disposable.